ci/unit/test_post_always_node.sh

```diff
@@ -5,7 +5,7 @@ set -uex
 cd "$DAOS_BASE"
 mkdir run_test.sh
 mkdir vm_test
-mv nlt-errors.json vm_test/
+mv nlt-errors.json vm_test/ || true
```

Comment:
This is a pretty broad exception that could paper over errors that shouldn't be.  What if `nlt-errors.json` actually did exist the but the `mv` failed for other reasons?

```suggestion
if [ -e nlt-errors.json ]; then
    mv nlt-errors.json vm_test/
fi
```